shell/client/admin/preinstalled-apps.html

```diff
@@ -0,0 +1,108 @@
+<template name="newAdminPreinstalledApps">
+  <h1>
+    <ul class="admin-breadcrumbs">
+      <li>{{#linkTo route="newAdminRoot"}}Admin{{/linkTo}}</li>
+      <li>Pre-installed Apps</li>
+    </ul>
+  </h1>
+
+  {{#if hasSuccess}}
+    {{#focusingSuccessBox}}
+      {{message}}
+    {{/focusingSuccessBox}}
+  {{/if}}
+  {{#if hasError}}
+    {{#focusingErrorBox}}
+      {{message}}
+    {{/focusingErrorBox}}
+  {{/if}}
+  <p>
+    Apps you configure on this page to be pre-installed will automatically be installed (show under /apps) for new users.
+  </p>
+  <form class="admin-preinstalled-apps">
+    <div class="form-group">
+      <label>
+        Productivity Suite
+      </label>
+      <span class="form-subtext">
+        We recommend pre-installing these core apps that are useful to most users.
+      </span>
+      <div class="productivity-suite" role="grid">
+        <div role="rowgroup">
+          {{#each productivityApps}}
+            {{> _appRow getRowData}}
+          {{/each}}
+        </div>
+      </div>
+    </div>
+
+    <!-- <div class="form-group">
+      <label>
+        System apps
+      </label>
+      <span class="form-subtext">
+        We recommend pre-installing these system apps that greatly enhance the abilities of Sandstorm.
+      </span>
+      <div class="productivity-suite" role="grid">
+        <div role="rowgroup">
+          {{#each systemApps}}
+            {{> _appRow getRowData}}
+          {{/each}}
+        </div>
+      </div>
+    </div> -->
+
+    <div class="form-group all-apps-group">
+      <label>
+        All apps on market
+      </label>
+      <span class="form-subtext">
+        The checked apps below will be installed by default for all new users.
+      </span>
+      <div class="all-apps" role="grid">
+        <div role="rowgroup">
+          {{#each allApps}}
+            {{> _appRow getRowData}}
+          {{/each}}
+        </div>
+        {{#if paginateApps}}
+          <div role="row" class="toggle-more">
+            <div role="gridcell">
+              <div role="button">
+                {{#if showAllApps}}
+                  <i class="up-arrow"></i>Show less apps
+                {{else}}
+                  <i class="down-arrow"></i>Show {{appCountMinusShown}} more apps
+                {{/if}}
+              </div>
+            </div>
+          </div>
+        {{/if}}
+      </div>
+    </div>
+
+    <div class="button-row">
+      <button type="submit" class="save" disabled="{{saveDisabled}}">Save</button>
+    </div>
+  </form>
+</template>
+
+<template name="_appRow">
```

Comment:
Mind documenting what a data context is expected to look like for this template?  e.g. which of the things below are expected to come from the data context, and which are actually template helpers?